drawing_agent/app/agent.py

```
@@ -115,19 +114,42 @@ async def run(self, path: str, question: str, thread_id: Optional[str] = None, p
+
+                if not heavy_analysis_text:
+                    logger.info(f"Кэш анализа не найден. Запуск нейросетей для {path}...")
+                    heavy_results = await self._run_heavy_operations(drawing_data["image_base64"])
+
+                    heavy_analysis_text = (
+                        f"--- РЕЗУЛЬТАТЫ ГЛУБОКОГО АНАЛИЗА ---\n"
+                        f"ДЕТЕКЦИЯ ОБЪЕКТОВ (YOLO):\n{heavy_results['yolo']}\n\n"
+                        f"ГЕОМЕТРИЯ (OpenCV):\n{heavy_results['geometry']}\n\n"
+                        f"ОТВЕРСТИЯ:\n{heavy_results['holes']}\n\n"
+                        f"ТАБЛИЦЫ:\n{heavy_results['tables']}\n\n"
+                        f"ПОЛНЫЙ ТЕКСТ:\n{heavy_results['full_ocr']}"
```

Ruff (0.15.12)

[warning] String contains ambiguous `Т` (CYRILLIC CAPITAL LETTER TE). Did you mean `T` (LATIN CAPITAL LETTER T)?

(RUF001)